helm_deploy/prison-api/values.yaml

```diff
@@ -24,6 +24,14 @@ generic-service:
     timeoutSeconds: 30
     failureThreshold: 15
 
+  resources:
+    requests:
+      cpu: 100m
+      memory: 640Mi
+    limits:
+      cpu: 2000m
+      memory: 2048Mi
```

Comment:
![Screenshot from 2022-03-09 10-20-25](https://user-images.githubusercontent.com/5099053/157422210-e8c7ee9c-9de6-4cf1-af36-34b790b94a7f.png)
Max recent usage is 1.26Gb, however it hasn't been using all the available heap allocated.  Since our `mx` setting is `1024` I've set this high enough not to hit the limit even if all available heap is allocated.
Nodes are `Standard DS3 v2 (4 vcpus, 14 GiB memory)` so should easily be able to cope as normally only 2 or 3 pods are installed to each node.